<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapDockWidget</class>
 <widget class="QDockWidget" name="MapDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>318</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Galaxy Map</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hyperdrive Class:</string>
        </property>
        <property name="buddy">
         <cstring>hyperdriveClass</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="hyperdriveClass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Trip Length (LY):</string>
      </property>
      <property name="buddy">
       <cstring>lightyears</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lightyears">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Slowest Time:</string>
      </property>
      <property name="buddy">
       <cstring>slowSpeed</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="slowSpeed">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Fastest Time:</string>
      </property>
      <property name="buddy">
       <cstring>fastSpeed</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fastSpeed">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Average Time:</string>
      </property>
      <property name="buddy">
       <cstring>avgSpeed</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="avgSpeed">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>94</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hyperdriveClass</tabstop>
  <tabstop>lightyears</tabstop>
  <tabstop>slowSpeed</tabstop>
  <tabstop>fastSpeed</tabstop>
  <tabstop>avgSpeed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
